eos-updater-flatpak-installer/eos-updater-flatpak-installer-fallback.service.in

```diff
@@ -0,0 +1,59 @@
+[Unit]
+Description=Endless OS Fallback Flatpak Installer
+Documentation=man:eos-updater-flatpak-installer(8)
+# /home is a symlink to /var/home; /var/home is a symlink to /sysroot/home. The
+# second symlink is created by systemd-tmpfiles. Since we use ProtectHome=yes,
+# we must explicitly order this unit after tmpfiles are created.
+Requires=local-fs.target eos-extra-settled.target systemd-tmpfiles-setup.service
+After=local-fs.target eos-extra-settled.target systemd-tmpfiles-setup.service
+ConditionKernelCommandLine=!eos-updater-disable
+DefaultDependencies=no
+Conflicts=shutdown.target
+
+# Unlike eos-updater-flatpak-installer.service, this service can run outside
+# of the normal update process.
```

Comment:
On the other hand we have `eos-updater-flatpak-installer-fallback.timer` starting the fallback service after 5 minutes and perhaps that's when it will succeed due to the network being up, so I suppose we're resigned to the flatpaks possibly being installed while the user is logged in.

Reply:
Yeah, this could run any time due to the timer, so I don’t think there’s any value in adding other restrictions.